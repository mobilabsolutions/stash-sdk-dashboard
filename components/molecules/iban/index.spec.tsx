import { testRender } from '../../../test_utils'
import IBAN from '.'

it('IBAN formater should format 16 size', () => {
  testRender(IBAN, {
    value: 'BE71096123456769'
  })
})

it('IBAN formater should format 22 size', () => {
  testRender(IBAN, {
    value: 'DE75512108001245126199'
  })
})
