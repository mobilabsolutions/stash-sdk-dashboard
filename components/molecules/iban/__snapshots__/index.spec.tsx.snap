// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IBAN formater should format 16 size 1`] = `
<span>
   BE71 0961 2345 6769
</span>
`;

exports[`IBAN formater should format 22 size 1`] = `
<span>
   DE75 5121 0800 1245 1261 99
</span>
`;
